packages/orbit-components/src/Popover/components/ContentWrapper.js

```diff
@@ -226,7 +228,7 @@ const PopoverContentWrapper = ({
     };
   }, []);
 
-  useClickOutside(popover, onClose);
+  useClickOutside(popover, onClose, !isTablet);
```

Comment:
Instead of extending the functionality of `useClickOutside` my idea was to simply wrap the handler you're passing.
```suggestion
  useClickOutside(popover, () => { if (isTablet) onClose(); });
```